src/query/storage/promremote/options.go

```diff
@@ -26,29 +26,46 @@ import (
 	"strings"
 
 	"github.com/uber-go/tally"
+	"go.uber.org/zap"
 
 	"github.com/m3db/m3/src/cmd/services/m3query/config"
+	"github.com/m3db/m3/src/query/storage/m3/storagemetadata"
 	xhttp "github.com/m3db/m3/src/x/net/http"
 )
 
 // NewOptions constructs Options based on the given config.
-func NewOptions(cfg *config.PrometheusRemoteBackendConfiguration, scope tally.Scope) (Options, error) {
+func NewOptions(
+	cfg *config.PrometheusRemoteBackendConfiguration,
+	scope tally.Scope,
+	logger *zap.Logger,
+) (Options, error) {
 	err := validateBackendConfiguration(cfg)
 	if err != nil {
 		return Options{}, err
 	}
 	endpoints := make([]EndpointOptions, 0, len(cfg.Endpoints))
 
 	for _, endpoint := range cfg.Endpoints {
-		endpointOptions := EndpointOptions{
-			name:    endpoint.Name,
-			address: endpoint.Address,
-		}
+		var (
+			attr = storagemetadata.Attributes{
+				MetricsType: storagemetadata.UnaggregatedMetricsType,
+			}
+			downsampleAll = false
```

Comment:
Hm I think we want downsampleAll to default to true as per defaultClusterNamespaceDownsampleOptions in query/storage/m3 package:
https://github.com/m3db/m3/blob/81c1f613883e6a0f1130d3b7927ec978c0bb74f1/src/query/storage/m3/cluster.go#L42-L44

Perhaps we export that var from package and reference it here?